### PR TITLE
fix(DMVP-0000): Created basic module for azure resource group.

### PR DESCRIPTION
**Pull request by viktoryathegreat**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+variable "rg_name" {
```
Comment (mrdntgrn) [inline variables.tf]: why not have variable names without "rg_" prefix,
name
location
tags

File: tests/basic/provider.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+provider "azurerm" {
+  skip_provider_registration = "true"
+
+  features {
+  }
+}
```
Comment (mrdntgrn) [inline tests/basic/provider.tf:6]: no tests provider described,
make sure `terraform test` works

File: outputs.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+output "rg_name" {
```
Comment (mrdntgrn) [inline outputs.tf]: same regarding prefix ''rg_" here, seems no special need to have it

File: locals.tf [hunk 4]
```diff
@@ -0,0 +1,4 @@
+locals {
+  default_tags = {}
```
Comment (mrdntgrn) [inline locals.tf]: are we going to use default tags?
Maybe we can have at least the controller_by( or created_by) tg with value terraform:
  default_tags = {created_by: "terraform"}

File: README.md [hunk 5]
```diff
@@ -1 +1,40 @@
-# terraform-azurerm-rg
+<!-- BEGIN_TF_DOCS -->
```
Comment (mrdntgrn) [inline README.md]: please check and include all automated check, validate tools we have(like checkov, terraform test, terrform fmt, terrform docs ....), see terraform_aws_modules or terraform_aws_repository,
you even can use terraform_aws_repository to automatically add this all

Comment (viktoryathegreat) [reply README.md]: done

File: outputs.tf (hunk 3 above)
Comment (viktoryathegreat) [reply outputs.tf]: done

File: tests/basic/provider.tf (hunk 2 above)
Comment (viktoryathegreat) [reply tests/basic/provider.tf:6]: Azure provider is described like this and it's enough for running the code, have tested, it works

File: README.md [hunk 6]
```diff
@@ -1 +1,96 @@
...
+| Name                                                                                                                        | Type     |
+| --------------------------------------------------------------------------------------------------------------------------- | -------- |
+| [azurerm_resource_group.rg](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group) | resource |
+
+
+## Inputs
+
+| Name                                                                  | Description                                                       | Type       | Default         | Required |
+| --------------------------------------------------------------------- | ----------------------------------------------------------------- | ---------- | --------------- | :------: |
+| <a name="input_rg_location"></a> [rg\_location](#input\_rg\_location) | The Azure Region where the Resource Group should exist.           | `string`   | `"West Europe"` |    no    |
```
Comment (mrdntgrn) [inline README.md]: please enable pre-commit hooks and make sure it works, it should fix input_rg_name=>input_name, ...

File: locals.tf [hunk 7]
```diff
@@ -0,0 +1,6 @@
+locals {
+  default_tags = {
+    ManagedBy : "Terraform"
+  }
+  all_tags = merge(local.default_tags, var.rg_tags)
```
Comment (mrdntgrn) [inline locals.tf]: it seems you have non existing variable named "rg_tags"

**Pull request merged**